package.json

```diff
@@ -6,7 +6,8 @@
   "scripts": {
     "serve": "vue-cli-service serve",
     "build": "vue-cli-service build",
-    "lint": "vue-cli-service lint"
+    "lint": "vue-cli-service lint",
+    "format": "vue-cli-service lint --no-fix --max-warnings 0"
```

Comment:
swap the names of these two commands (npm lint shouldnt fix while npm format will fix)